cmake/setup_write_config.cmake

```diff
@@ -211,6 +216,7 @@ FOREACH(_feature ${_deal_ii_features_sorted})
     #
     IF(NOT "${${_feature}_DIR}" STREQUAL "")
       _detailed("#            ${_feature}_DIR = ${${_feature}_DIR}\n")
+      _featurelog("SET(${_feature}_DIR \"${${_feature}_DIR}\")\n")
```

Comment:
What about adding a `DEAL_II_` prefix as you suggested?

Thinking about it after a couple of hours of sleep, that would be cleaner as we avoid name clashes.